docs/references/nextjs/with-clerk-middleware.mdx

```diff
@@ -5,7 +5,7 @@ description: The withClerkMiddleware wrapper allows Clerk to access session data
 
 # `withClerkMiddleware()`
 
-<Callout type="warning">
+<Callout type="danger">
   `withClerkMiddleware()` is Clerk's previous middleware API and will soon be deprecated. Please use [`authMiddleware()`](/docs/references/nextjs/auth-middleware) going forward.
```

Comment:
Can we give something more concrete than "soon?" A date, a release number, or just "an upcoming release"?